ecosystem/sep-0006.md

```diff
@@ -131,7 +131,7 @@ Request Parameters:
 
 Name | Type | Description
 -----|------|------------
-`asset_code` | string | The code of the asset the user wants to deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
+`asset_code` | string | The code of the off-chain asset the user wants to deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
```

Comment:
This is actually the on-chain asset, as specified in the `/info` description:

> The JSON object contains an entry for each Stellar asset that the anchor supports for SEP6 deposit and/or withdrawal.

Reply:
Reverted this change in https://github.com/stellar/stellar-protocol/pull/972/commits/2672aba7e7d62e3cc830f7b5cc02b6c6326915bf.